src/kibana/partials/saved_object_finder.html

```diff
@@ -1,30 +1,36 @@
 <form role="form">
   <div class="form-group finder-form">
     <div class="finder-form-options">
-      <a class="small" ng-click="manageObject(type)">manage {{type}}</a>
+      <a class="small" ng-click="finder.manageObjects(finder.objectType.name)">manage {{finder.objectType.name}}</a>
     </div>
     <div class="clearfix visible-xs-block"></div>
-    <input
-      input-focus
-      ng-model="filter"
-      ng-attr-placeholder="{{noun}} Filter"
-      class="form-control"
-      name="filter"
-      type="text">
-      <span class="finder-hit-count"><strong>{{hitCount}}</strong> {{type}}</span>
+    <input input-focus
+           ng-model="filter"
```

Comment:
![ditto](https://cloud.githubusercontent.com/assets/1329312/8174878/08bd9ef0-1397-11e5-815f-46b635274585.gif)

the multi-line-ness is appreciated but lets keep it two-space indented